components/grid-col/src/index.js

```diff
@@ -20,33 +25,31 @@ const StyledColumn = styled('div')(
       },
     },
   },
-  ({ hideContent }) => ({
-    textIndent: hideContent ? '-999em' : undefined,
-    backgroundColor: hideContent ? '#7DADD3' : undefined,
-    backgroundImage: hideContent
-      ? 'repeating-linear-gradient(180deg, #7DADD3, #7DADD3 15px, #B7CFE1 15px, #B7CFE1 30px)'
-      : undefined,
-  }),
-  ({ columnOneThird }) => ({
-    [MEDIA_QUERIES.LARGESCREEN]: {
-      width: columnOneThird ? '33.3333%' : undefined,
-    },
-  }),
-  ({ columnTwoThirds }) => ({
-    [MEDIA_QUERIES.LARGESCREEN]: {
-      width: columnTwoThirds ? '66.6667%' : undefined,
-    },
-  }),
-  ({ columnOneQuarter }) => ({
-    [MEDIA_QUERIES.LARGESCREEN]: {
-      width: columnOneQuarter ? '25%' : undefined,
-    },
-  }),
-  ({ columnOneHalf }) => ({
-    [MEDIA_QUERIES.LARGESCREEN]: {
-      width: columnOneHalf ? '50%' : undefined,
-    },
-  }),
+  ({ hideContent }) => {
+    if (!hideContent) { return false; }
+    return ({
+      textIndent: '-999em',
+      backgroundColor: '#7DADD3',
+      backgroundImage: 'repeating-linear-gradient(180deg, #7DADD3, #7DADD3 15px, #B7CFE1 15px, #B7CFE1 30px)',
+    });
+  },
+  (...args) => {
```

Comment:
Rather than `...args` here this could have just been `props` and instead of `args[0]` below that could have been `prope` too

Would be a little easier to understand 

You’re not looking at any other argument being passed in so the spread isn’t needed